Entered training script...
Python executable: /home/umflint.edu/koernerg/excelformer/ExcelFormer/env/bin/python
Python path: ['/home/umflint.edu/koernerg/excelformer/ExcelFormer', '/home/umflint.edu/koernerg/.pyenv/versions/3.8.18/lib/python38.zip', '/home/umflint.edu/koernerg/.pyenv/versions/3.8.18/lib/python3.8', '/home/umflint.edu/koernerg/.pyenv/versions/3.8.18/lib/python3.8/lib-dynload', '/home/umflint.edu/koernerg/excelformer/ExcelFormer/env/lib/python3.8/site-packages', '/home/umflint.edu/koernerg/excelformer/ExcelFormer/env/lib/python3.8/site-packages/setuptools/_vendor']
LD_LIBRARY_PATH: Not set
CUDA_HOME: Not set
Torch version: 1.11.0+cu113
Torch CUDA version: 11.3
CUDA available: True
CUDA device count: 1
nvidia-smi output:
 Wed Mar 12 17:58:21 2025       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 560.35.03              Driver Version: 560.35.03      CUDA Version: 12.6     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA GeForce RTX 4090        Off |   00000000:01:00.0 Off |                  Off |
|  0%   34C    P8              2W /  450W |      18MiB /  24564MiB |      0%      Default |
|                                         |                        |                  N/A |
+-----------------------------------------+------------------------+----------------------+
                                                                                         
+-----------------------------------------------------------------------------------------+
| Processes:                                                                              |
|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
|        ID   ID                                                               Usage      |
|=========================================================================================|
|    0   N/A  N/A      2482      G   /usr/lib/xorg/Xorg                              4MiB |
+-----------------------------------------------------------------------------------------+

Passed imports...
CUDA available: True
CUDA device count: 1
Current CUDA device: 0
CUDA device name: NVIDIA GeForce RTX 4090
Loading data from: /home/umflint.edu/koernerg/excelformer/ExcelFormer/DATA/android_security
Initial DataFrame shape: (870514, 50)
Loading top 25 XGBoost features: ['ContentRating' 'Genre' 'days_since_last_update' 'developer_address'
 'CurrentVersion' 'AndroidVersion' 'LastUpdated' 'TwoStarRatings'
 'ThreeStarRatings' 'max_downloads_log' 'DeveloperCategory'
 'lowest_android_version' 'LOCATION' 'isSpamming' 'STORAGE'
 'OneStarRatings' 'PHONE' 'highest_android_version' 'intent'
 'FiveStarRatings' 'Unnamed: 0' 'privacy_policy_link' 'developer_website'
 'LenWhatsNew' 'FourStarRatings']
Shape after dropping NaNs: (724160, 49)
Traceback (most recent call last):
  File "run_default_config_excel.py", line 155, in <module>
    dataset = build_dataset(DATA / args.dataset, transformation, T_cache)
  File "/home/umflint.edu/koernerg/excelformer/ExcelFormer/lib/data.py", line 443, in build_dataset
    dataset = Dataset.from_dir(path)
  File "/home/umflint.edu/koernerg/excelformer/ExcelFormer/lib/data.py", line 91, in from_dir
    df = df[features_to_keep]
  File "/home/umflint.edu/koernerg/excelformer/ExcelFormer/env/lib/python3.8/site-packages/pandas/core/frame.py", line 3464, in __getitem__
    indexer = self.loc._get_listlike_indexer(key, axis=1)[1]
  File "/home/umflint.edu/koernerg/excelformer/ExcelFormer/env/lib/python3.8/site-packages/pandas/core/indexing.py", line 1314, in _get_listlike_indexer
    self._validate_read_indexer(keyarr, indexer, axis)
  File "/home/umflint.edu/koernerg/excelformer/ExcelFormer/env/lib/python3.8/site-packages/pandas/core/indexing.py", line 1377, in _validate_read_indexer
    raise KeyError(f"{not_found} not in index")
KeyError: "['Unnamed: 0'] not in index"
